mappings/net/minecraft/world/level/LevelProperties.mapping

```diff
@@ -43,15 +43,20 @@ CLASS net/minecraft/class_31 net/minecraft/world/level/LevelProperties
 	FIELD field_190 raining Z
 	FIELD field_191 scheduledEvents Lnet/minecraft/class_236;
 	FIELD field_192 rainTime I
-	FIELD field_193 worldData Ljava/util/Map;
+	FIELD field_193 worldNbt Ljava/util/Map;
```

Comment:
Does this rename really make sense here ?

Reply:
it seems strange that it is referred to as "data" here when everywhere else we call nbt what it is...nbt.